contrib/sssd.sysusers

```diff
@@ -0,0 +1 @@
+u     sssd   -   "User for sssd"     /               /sbin/nologin
```

Comment:
Don't we need a `g` group line entry here also?

Reply:
The `g` is implied with systemd-sysusers.

https://www.freedesktop.org/software/systemd/man/sysusers.d.html#u